### PR TITLE
SUPPORT.MD proposal

### PR DESCRIPTION
**Pull request by jeffwilcox**
The CSS (Microsoft Customer Service and Support) team has a process
for providing support to some products or projects. To help improve
the experience for such cases, this update proposes a new SUPPORT.MD
file to be placed in the default repo templates with information for
maintainers to update and publish.

For questions in the proposal, please reach out to JHornick or comment
on this original pull request.

This PR is originally targeting the `mit` (the standard Microsoft new project repo type).

### CODE REVIEW

File: projections/mit/SUPPORT.md
Comment (JamieMagee) [inline projections/mit/SUPPORT.md:21]:
```suggestion
CHANNEL. WHERE WILL YOU HELP PEOPLE?].**

```

**Comment by johngossman** [comment]
lgtm

File: projections/mit/SUPPORT.md
Comment (andschwa) [inline projections/mit/SUPPORT.md:25]: What's this mean "**PROJECT or PROJECT**"?

File: projections/mit/SUPPORT.md
Comment (thomastaylor312) [inline projections/mit/SUPPORT.md:15]: Can we auto generate a link to the issues page of the new repo? Basically make it so "Github Issues" and "Issue" (down below) are hyperlinked to the issues page

Comment (jeffwilcox) [reply projections/mit/SUPPORT.md:15]: Wish so, but it's not currently a feature/capability of the system.

**Review by thomastaylor312 — APPROVED** [review]

File: projections/mit/SUPPORT.md
Comment (jeffwilcox) [reply projections/mit/SUPPORT.md:25]: I think the guidance from @jhornick-ms is that they want the maintainer to clarify if it's a product or a project... products might be something like Cosmos DB vs a project like "some Cosmos integration for my product", maybe?

I'll let @jhornick-ms comment

File: projections/mit/SUPPORT.md
Comment (jeffwilcox) [reply projections/mit/SUPPORT.md]: fixed

File: projections/mit/SUPPORT.md
Comment (andschwa) [reply projections/mit/SUPPORT.md:25]: I wondered, in which case this should probably be "PROJECT or PRODUCT" (unless my eyes are deceiving me, it says "project" twice).

Comment (jhornick-ms) [reply projections/mit/SUPPORT.md:25]: @andschwa - @jeffwilcox  is right - this verbiage was intended to allow the repo owners to indicate whether this is a project, or something that rises to the level of "product" which is more likely - but not required - to carry official support.  There are entities which differentiate between the two, so we started with this in an attempt to embrace those terms.  If it's confusing, we could alter to "Support for the contents of this repo is limited to the resources listed above."

Comment (bureado) [reply projections/mit/SUPPORT.md:25]: It's a typo.

**Pull request merged**
